src/Data/Quaternion/Rotation.purs

```diff
@@ -107,3 +112,24 @@ act (Rotation p) v =
   unsafePartial $ case Vec.toArray v of
     [x, y, z] ->
       vectorPart (Quaternion zero x y z `conjugateBy` p)
+
+normalize :: Rotation Number -> Rotation Number
```

Comment:
Can you explain why this is needed? A `Rotation` should already be normalized, ideally.